pkg/schema/thema_schema.go

```diff
@@ -0,0 +1,61 @@
+package schema
```

Comment:
This whole file looks generic, i'm guessing we should be able to move basically all of it (eventually) to github.com/grafana/thema/crd

🎉 🎉

Reply:
Yup, makes sense to me 👍